evm/src/Transceiver/WormholeTransceiver/WormholeTransceiver.sol

```diff
@@ -194,14 +195,24 @@ contract WormholeTransceiver is
             parseWormholeTransceiverInstruction(instruction.payload);
 
         if (!weIns.shouldSkipRelayerSend && _shouldRelayViaStandardRelaying(recipientChain)) {
+            // NOTE: standard relaying supports refunds. The amount to be refunded will be sent
+            // to a refundAddress specified by the client.
```

Comment:
Worth being explicit in this comment that this is to the address on the destination chain. Also worth adding to the documentation I think